public/apps/configuration/panels/tenant-list/test/tenant-list.test.tsx

```diff
@@ -131,7 +131,7 @@ describe('Tenant list', () => {
         config={config1 as any}
       />
     );
-    expect(component.find(TenantInstructionView).length).toBe(1);
+    expect(component.find(TenantInstructionView).length).toBe(0);
```

Comment:
For my understanding, why we change the test case here?

Reply:
We changes the way manage tab would look when multitenancy.enabled is false. Hence the result will change. This is now same as before we merged our changes for dynamic tenancy config.